ftests/nuxeo-platform-explorer-ftests/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Log4j Configuration - For more configuration information and examples see
+  the Jakarta Log4j - website: http://jakarta.apache.org/log4j
+
+  Duplicated from nuxeo-distribution-tomcat/src/main/resources/common/lib/log4j.xml
+-->
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/"
```

Comment:
ah! i actually copy/pasted other setups... see https://github.com/nuxeo/nuxeo/blob/master/ftests/nuxeo-jsf-to-web-ui-ftests/src/test/resources/log4j.xml for instance.
but i'll fix this one

Reply:
I created [NXP-28932](https://jira.nuxeo.com/browse/NXP-28932).

Reply:
i cannot actually rename it cleanly without fixing every package because the file is mentioned in the ftests pom.xml, so i left it as is so that it's handled with NXP-28932. I hope it's ok!

Reply:
(done)